Source/WebKit/NetworkProcess/cocoa/AppStoreDaemonSPI.h

```diff
@@ -45,10 +48,12 @@ typedef NS_ENUM(NSInteger, ASDInstallWebAttributionContext) {
 @property (nonatomic, assign) ASDInstallWebAttributionContext attributionContext;
 @end
 
-@interface ASDInstallAttribution : NSObject
-+ (instancetype)sharedInstance;
-- (void)addInstallWebAttributionParamsWithConfig:(ASDInstallWebAttributionParamsConfig *)config completionHandler:(void (^)(NSError *error))completionHandler;
+#if HAVE(ASD_INSTALL_WEB_ATTRIBUTION_SERVICE)
```

Comment:
Won't we regress on platforms that don't have this service?

Reply:
That platform is iOS 16, which will never see this change.

Reply:
Thanks!